x/axelarnet/client/rest/query.go

```diff
@@ -0,0 +1,42 @@
+package rest
```

Comment:
this way to define the rest server is deprecated, delete file completely

Reply:
done